index.html

```diff
@@ -4253,9 +4100,175 @@ <h3>Parsing content</h3>
+        Browsers may ignore this rule for development purposes only.
+      </p>
+    </section>
+    <section> <h4>Visible document</h4>
+      <p>
+        Web NFC functionality is allowed only for the {{Document}} of the
+        <a>top-level browsing context</a>, which must be
+        <dfn data-cite="PAGE-VISIBILITY#dom-visibilitystate-visible">visible</dfn>.
+      </p>
+      <p>
+        This also means that UAs should block access to the NFC radio if
+        the display is off or the device is locked.
+        For backgrounded web pages, receiving and pushing <a>NFC content</a>
+        must be <a id="#nfc-suspended">suspended</a>.
+      </p>
+    </section>
+    <section> <h4>Permissions controls</h4>
+      <p>
+        Making an <a>NFC tag</a> read-only must <a>obtain permission</a>, or
+        otherwise fail.
+      </p>
+      <p>
+        Setting up listeners for reading <a>NFC content</a> should
+        <a>obtain permission</a>.
+      </p>
+      <p>
+        The process of reading an <a>NDEF message</a> does not need to
+        <a>obtain permission</a>.
+      </p>
+      <p>
+        Pushing <a>NFC content</a> to an <a>NFC peer</a> does not need to
+        <a>obtain permission</a>, but the other rules in this section apply.
+        See the [[[#writing-or-pushing-content]]] section.
+      </p>
+      <p>
+        Pushing an <a>NDEF message</a> to an <a>NFC tag</a> does not need to
+        <a>obtain permission</a>, if the existing <a>NDEF message</a> only
+        contains <a>NDEF record</a>s without <a>ID field</a>s, or with <a>
+        ID field</a>s matching the [=host/registrable domain=] of the
+        <a>current settings object</a>'s origin.
+        Otherwise the UA must <a>obtain permission</a> for pushing
+        <a>NFC content</a> which overwrites existing information.
+        See also the [[[#writing-or-pushing-content]]] section.
+      </p>
+      <p>
+        Since all local content that a web page has access to can be shared with
+        NFC, the user needs to be clearly aware about the permissions granted
+        to the web page using Web NFC.
+      </p>
+    </section>
+    <section> <h4>Store site URL as record identifier when writing data</h4>
+      <p>
+        When pushing an <a>NDEF message</a>, the [= host/registrable domain =],
+        <a data-cite="dom#concept-host-serializer">serialized</a>,
+        of the <a>current settings object</a> must be stored as the <a>record
+        identifier</a> in each top-level <a>NDEF Record</a>.
+      </p>
+    </section>
+    <section> <h4>Warn risk of physical location leak</h4>
+      <p>
+        When listening for and pushing <a>NFC content</a>,
+        the UA may warn the user that the given <a>origin</a> may be able to
+        infer physical location.
+      </p>
+    </section>
+    <section> <h4>Restrict automatic handling</h4>
+      <p>
+        The payload data on <a>NFC content</a> is untrusted, and must not be used
```

Comment:
But people can use signature records to make it trusted

Reply:
> a malicious third party could delete the signature record from the NDEF message or
attach a new signature record to prevent the user from noticing any malicious change of content.

Reply:
But with the right policies in place it should be indeed fine (trusted). 
Anyway, content changes come in a next PR.